### DISCUSSION TITLE
Directory merge subpath for SSD and Spninning Disk

### DISCUSSION THREAD

**Discussion by tyjtyj** [trapexit/mergerfs#1258] [Q&A] [answered]
I need some help understanding epmfs policy. 

I have 3 drives ssd, mechA and mechB with dir path as below where mechA/B is spinning drive. I would like to fill up SSD with download and use mechA as backup increase ssd runs out of space. mechB will be backup of mechA incase mechA out of space
SSD 100GB free
/nasdisk/download
/folder1
/file1
MechA 4TB free
/nasdisk/folder1
/nasdisk/folder2
/nasdisk/folder3
/nasdisk/file1
/nasdisk/file2
/nasdisk/file3
MechB 2tb free
/nasdisk/folder11
/nasdisk/foldee12
/nasdisk/folder13
/nasdisk/file11
/nasdisk/file12
/nasdisk/file13

how should i mount it to avoid fillup SSD when creating files and folder on /nasdisk ? 
is this the correct setting ?
/etc/fstab
`/mnt/mechA:/mnt/ssd:/mnt/mechB /mnt/hdd fuse.mergerfs minfreespace=10G,cache.files=partial,dropcacheonclose=true,category.create=epmfs,x-systemd.mount-timeout=1min 0 0`

or (not the position of /mnt/ssd)
`/mnt/mechA:/mnt/mechB :/mnt/ssd /mnt/hdd fuse.mergerfs minfreespace=10G,cache.files=partial,dropcacheonclose=true,category.create=epmfs,x-systemd.mount-timeout=1min 0 0`

**Comment by trapexit** [comment]
You can't use `epmfs` and do anything like that. If you want to prioritize the drives by order then just put them in the relevant order and use `ff` with `minfreespace` set accordingly.

**Reply by tyjtyj** [reply]
Thanks for reply. 

If I proceed with 
/mnt/mechA:/mnt/mechB:/mnt/ssd /mnt/hdd fuse.mergerfs minfreespace=10G,cache.files=partial,dropcacheonclose=true,category.create=epmfs,x-systemd.mount-timeout=1min 0 0

what happen when ssd filled up until 10GB free.. will it create a folder in /mnt/mechA/nasdisk/download? 
I am ok to manually delete the folder /mnt/mechA/nasdisk/download and let it fall back to /mnt/ssd/nasdisk/download

**Reply by trapexit** [reply, marked as answer]
As I said `empfs` won't do what you want. If you want to always prioritize the ssd then you should put it first in the list and use `ff` create policy.

Don't think about creating directories. mergerfs policies choose the branch to work on and then the branch is copied as needed. Read the section in the docs about policies and find the combination that does what you're looking for.
